app/src/main/java/ch/epfl/sdp/GeoPoint.java

```diff
@@ -19,8 +19,11 @@
      */
     public static final double EARTH_RADIUS = 6371000;
 
-    private final double longitude;
-    private final double latitude;
+    public double longitude;
+    public double latitude;
+
+    public GeoPoint() {
```

Comment:
since Geopoint is part of Player class and I want to store the Player class into cloud firebase, either I need to change the field to be public or I need to have setter/getter for each field. Since  changing to public more convenient for me, here I change to public directly. But later I can use setter/getter if we prefer.

Reply:
Changed to getter/setter